twake/frontend/src/app/services/RouterService.ts

```diff
@@ -179,7 +196,7 @@ class RouterServices extends Observable {
     });
 
     //Retrocompatibility with old code
-    state.companyId = Collections.get('workspaces').find(state.workspaceId)?.group?.id || '';
+    state.companyId = Collections.get('workspaces').find(state.workspaceId) ? Collections.get('workspaces').find(state.workspaceId).company_id : '';
```

Comment:
Is this better than Collections.get('workspaces').find(state.workspaceId)?.company_id || ''